api.swagger.yaml

```
@@ -3130,14 +3130,14 @@ components:
           type: string
           example: image/jpeg
         alt_text:
-          description: Alt text for an image.
+          description: Alt text for an image. Only required and returned if the type is an image.
           type: string
         image_height:
-          description: The height of the image in pixels. Only returned if the type is an image.
+          description: The height of the image in pixels. Only required and returned if the type is an image.
```

Hmm on second thoughts... what does "required" mean here? Don't we have a separate schema for writes, just below? (and it doesn't include the image dimensions, because those are computed from the image)

---

required means it has to be in the response when the type of file is an image.

---

in that case i would rephrase it:
```suggestion
          description: The height of the image in pixels. Returned if and only if the type is an image.
```